api-reference/beta/resources/alert.md

```diff
@@ -0,0 +1,120 @@
+|feedback|string|Analyst feedback on the alert. Possible values are: `unknown`, `truePositive`, `falsePositive`, `benignPositive`. ([Supports Update](../api/alert_update.md))|
+|fileStates|[fileSecurityState](filesecuritystate.md) collection|Security-related stateful information generated by the provider about the file(s) related to this alert.|
+|hostStates|[hostSecurityState](hostsecuritystate.md) collection|Security-related stateful information generated by the provider about the host(s) related to this alert.|
+|id|String *|Provider-generated GUID/unique identifier. (Read-only)|
+|lastModifiedDateTime|DateTimeOffset|Time at which the alert entity was last modified. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`.|
+|malwareStates|[malwareState](malwarestate.md) collection|Threat Intelligence pertaining to malware related to this alert.|
+|malwareWasRunning|Boolean|Was the detected file (malware/vulnerability) running at the time of detection or was it detected at rest on disk?|
+|networkConnections|[networkConnection](networkconnection.md) collection|Security-related stateful information generated by the provider about the network connection(s) related to this alert.|
+|processes|[process](process.md) collection|Security-related stateful information generated by the provider about the process or processes related to this alert.|
+|recommendedActions|String collection|Vendor/Provider recommended action(s) to take as a result of the alert (e.g. isolate machine, enforce2FA, reimage host, etc.).|
+|riskScore|String|Provider generated/calculated risk score of the alert.|
+|severity|String *|Alert severity - set by vendor/provider.|
+|sourceMaterials|String collection|Hyperlinks (URIs) to the source material related to the alert, e.g. provider's user interface for alerts or log search, etc.|
+|status|String *|Alert lifecycle status (stage). Possible values are: `unknown`, `newAlert`, `inProgress`, `resolved`. ([Supports Update](../api/alert_update.md))|
+|tags|String collection|User-definable labels that can be applied to an alert and can serve as filter conditions (e.g. "HVA", "SAW", etc.) ([Supports Update](../api/alert_update.md)).|
+|title|String *|Alert Title.|
+|triggers|[alertTrigger](alerttrigger.md) collection|Security-related information about the specific attribute(s) that triggered the alert (attribute(s) appearing in the alert). Alerts may contain information about multiple users, hosts, files, ip addresses, etc. This field is used to indicate which of these actually triggered the alert generation.|

```

Comment:
Change "may" to "might" (may implies permission). Revised second sentence as "...multiple users, hosts, files, or IP addresses."